### ISSUE TITLE
building special silva : No such file ‘tax_slv_ssu_nr_132.acc_taxid

### ISSUE THREAD

**Issue by kmavrommatis** [DerrickWood/kraken2#43]
Hi,
This could be related to #8 
When applying the change suggested in the `16S_silva_installation.sh` script
the kraken2-build command works.

```
bin/kraken2-build --special silva --db $PWD/silva --threads $(nproc)
```

However, when I run `kraken2-inspect` with the database that was created in the previous step the only named node I get is root. All other nodes are not named.

```
bin/kraken2-inspect --db /db/silva/  | head -100:$PWD -u $(id -u)
# Database options: nucleotide db, k = 35, l = 31
# Spaced mask = 11111111111111111111111111111111111111110011001100110011001100
# Toggle mask = 1110001101111110001010001100010000100111000110110101101000101101
# Total taxonomy nodes: 13528
# Table size: 26477341
# Table capacity: 37889828
100.00% 26477341        13956   R       1       root
 73.25% 19395500        1121078 R1      3
 18.61% 4928033 76036   R2      2375
  9.73% 2575341 146135  R3      3303
  2.42% 640458  33886   R4      26341
  1.43% 377766  42528   R5      26352
  0.16% 43095   43095   R6      26368
  0.07% 18838   18838   R6      26414
  0.06% 16298   16298   R6      26384
  0.06% 16233   16233   R6      26471
  0.06% 15816   15816   R6      26355
  0.06% 14829   14829   R6      26361
  0.04% 11854   11854   R6      26358
  

```

Consequently, (which may be an unrelated issue), when I try to classify sequences with kraken2 I get the results with taxid but not names of taxa eg:

```
bin/kraken2 --db /db/silva --report $PWD/report.txt --classified-out $PWD/clas.txt --output $PWD/out.txt  --use-names $PWD/00A00044_Ext2_Rep2_S172_L001_R1_001.fastq.gz
118679 sequences (35.72 Mbp) processed in 3.534s (2014.7 Kseq/m, 606.43 Mbp/m).
  118140 sequences classified (99.55%)
  539 sequences unclassified (0.45%)
kmavrommatis@ip-10-112-17-141: Tue Aug 21 22:09:02 microbiome (0)
$more out.txt
C	M04141:136:000000000-B96YR:1:1101:22984:1662	918	301	912:1 3:3 912:1 3:36 913:5 918:1 3:1 912:3 918:9 912:5 3:40 918:2 913:5 918:19 912:5 918:2 3:3 918:3 3:49 913:2 912:5 3:1 912:1 918:25 913:1 3:2 918:2 3:1 0:5 3:1 918:5 3:2 913:7 918:1 0:3 913:5 0:5
C	M04141:136:000000000-B96YR:1:1101:19609:1681	1863	301	3:43 1863:27 3:17 1863:5 3:15 1863:28 3:1 1863:8 3:64 1863:11 3:38 1:5 0:5
C	M04141:136:000000000-B96YR:1:1101:14361:1737	1863	301	3:26 1863:1 3:16 1863:44 3:4 1863:50 3:5 1:1 3:42 1:1 3:72 1:2 0:3
C	M04141:136:000000000-B96YR:1:1101:22479:1741	918	301	3:51 913:5 3:1 918:3 912:5 3:36 918:34 3:3 918:8 3:54 918:2 912:5 918:16 3:10 918:5 3:1 918:5 3:2 913:5 0:6 913:5 1780:2 0:3
C	M04141:136:000000000-B96YR:1:1101:9321:1752	24660	301	3:131 913:10 3:6 913:4 3:23 913:27 912:1 24660:22 913:3 3:35 0:5
C	M04141:136:000000000-B96YR:1:1101:11659:1762	11328	301	3:59 1863:1 3:5 1863:16 3:10 1863:2 3:4 1863:10 1987:5 1863:4 1987:15 3:83 1863:5 1987:3 1863:7 1672:3 1863:9 3:5 1:7 3:1 1:3 11328:5 3:2 0:3
C	M04141:136:000000000-B96YR:1:1101:9249:1764	1987	301	3:59 1863:1 3:5 1863:16 3:10 1863:2 3:4 1863:10 1987:5 1863:4 1987:15 1672:5 3:3 1672:2 3:105 1:7 0:1 1:3 0:10
C	M04141:136:000000000-B96YR:1:1101:10307:1811	1863	301	3:46 1863:36 3:9 1863:55 1:1 3:42 1:1 3:63 0:1 3:3 0:5 2588:2 0:3
``` 

For comparison, when the same commands are run on a kraken2 database created from refseq all results include the taxa names.

Any advice?

Thanks

**Comment by DerrickWood** [comment]
I believe the issues you're referencing were fixed in https://github.com/DerrickWood/kraken2/commit/ee24dd32626e77324b0c6f3682e0a9d00c4b7917 - is your code from an earlier commit?

**Comment by DerrickWood** [comment]
Hi @kmavrommatis - is this issue not fixed with the 2.0.7 version (and a redownload/build of the database)?

**Comment by kmavrommatis** [comment]
The issue is resolved with the latest version (2.0.7-beta)
Thanks
